capi/geos_c.cpp

```diff
@@ -25,6 +25,7 @@
 #include <geos/operation/overlay/OverlayOp.h>
 #include <geos/operation/union/CascadedPolygonUnion.h>
 #include <geos/algorithm/distance/DiscreteHausdorffDistance.h>
+#include <geos/algorithm/distance/DiscreteDTWDistance.h>
```

Comment:
I'm curious whether/why this needs to be included here; you're clearly following by example which is fine by me. More of a question for maintainers.